parse.go

```diff
@@ -227,6 +228,30 @@ func parseListLine(line string, now time.Time, loc *time.Location) (*Entry, erro
 	return nil, errUnsupportedListLine
 }
 
+func parseQuotas(line string) (entries map[string]string, err error) {
+	entries = make(map[string]string)
+
+	// Regex Sample https://regex101.com/r/lv2nWj/1/
+	var re = regexp.MustCompile(`(?m)^(\s*)([a-zA-Z\-\_0-9\\\/\.\s\[\]\s]*):(\s*)([a-zA-Z\-\_0-9\\\/\.]*)$`)
+	var link = regexp.MustCompile(`(^[A-Za-z])|(_|\s)([A-Za-z])`)
+	for i, match := range re.FindAllStringSubmatch(line, -1) {
+		if i == 0 {
+			continue // Skip the intro line.
+		}
+		if match[2] != "" {
+
+			key := link.ReplaceAllStringFunc(match[2], func(s string) string {
```

Comment:
What is the advantage in removing whitespace?

Reply:
Is there an advantage to keeping Whitespaces?
Introducing whitespaces IMO can create confusion and consume more memory (realistically nothing noticeable).

In either case, there is no standard for how the specific `SITE` FTP command is formatted and by doing this helps tighten the expected results. (ref: [RFC959](https://tools.ietf.org/html/rfc959#page-33), [RFC1123](https://tools.ietf.org/html/rfc1123#page-32), [Wiki](https://en.wikipedia.org/wiki/List_of_FTP_commands))

#### RFC1123
> ...the SITE command for non-standard features, rather than invent new private commands or unstandardized extensions to existing commands.

Reply:
Why not `strings.Trim(line, " ")` ?